backend/models/user.py

```diff
@@ -26,14 +25,14 @@ def json(self) -> dict:
 
     def add_user(self):
         """
-        add new plan into the db. It generates the plan uid.
+        add new user into the db. It generates the user uid.
         :return: None
         """
         try:
-            update_time, us_ref = db.collection(u'users').add({u'username': self.username, u'joined': self.joined})
+            update_time, us_ref = db.collection(u'users').add({u'username': self.username, u'joined': self.joined}, timeout=90)
             self.uuid = us_ref.id
         except Exception as e:
-            raise UserDBAddingError(self) from e
+            raise e
```

Comment:
would be greate to keep the exception UserDBAddingError() from e. ( I understand that you have temporally changed it in order to see what is really going on)

Reply:
I've fixed. Nice catch!